website/docs/getting-started/faq.md

```diff
@@ -17,6 +17,10 @@ Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}
 
 Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Appsmith is an open-source platform and can be fully audited and deployed on-premise to ensure none of your data leaves your VPC.
 
+## Do you have a bounty program for reporting security vulnerabilities?
```

Comment:
I think - This question should be added as below:

Q: Do you have a bounty program for reporting security vulnerabilities?
A: No, we do not currently have a bounty program for reporting security vulnerabilities. However, we are always grateful for any information that can help us improve the security of our systems and protect our users' data.

Q: How do I report a security vulnerability?
A: If you believe you have discovered a security vulnerability, please email our security team at security@appsmith.com with a description of the issue and any relevant details. We will review your report and take appropriate action to address the issue.

Reply:
Agree with @jnikhila on this. Just a small modification to the content

Q: How do I report a security vulnerability?
A: We appreciate any information that can help us improve the security of our systems and protect our users' data. If you believe you have discovered a security vulnerability, please email our security team at [security@appsmith.com](mailto:security@appsmith.com) with a description of the issue and any relevant details. We will review your report and take appropriate action to address the issue.